packages/components/addon/components/hds/accordion/item/index.hbs

```diff
@@ -0,0 +1,25 @@
+<Hds::DisclosurePrimitive class={{this.classNames}} @isOpen={{@isOpen}} ...attributes>
+  <:toggle as |t|>
+    <div class="hds-accordion-item__toggle">
+      <Hds::Accordion::Item::Button
+        @isOpen={{t.isOpen}}
+        @onClickToggle={{t.onClickToggle}}
+        @contentId={{this.contentId}}
+        @ariaLabel={{this.ariaLabel}}
+        @isInteractive={{@isInteractive}}
+      />
+
+      <div
+        class="hds-accordion-item__toggle-content hds-typography-display-200 hds-foreground-strong hds-font-weight-semibold"
```

Comment:
while doing some tests, I realized this element is not 100% in width.
I _think_ it should, otherwise consumers will not be able to achieve something like this:
<img width="526" alt="screenshot_2818" src="https://github.com/hashicorp/design-system/assets/686239/32eff437-163c-4cd1-a780-e0083a06c98e">

Reply:
It seems to be full width to me but I can add an explicit width style.

Reply:
Sorry I was looking at the wrong element... Re-checking.

Reply:
Alright, pushed a fix for the correct element this time.